packages/entities/entities-plugins/src/components/free-form/Datakit/flow-editor/node-forms/NodeFormCall.vue

```diff
@@ -1,5 +1,6 @@
 <template>
   <Form
+    :key="`${formRefreshKey}`"
```

Comment:
```suggestion
    :key="formRefreshKey"
```

Same below.

And what is this key used for?